### PR TITLE
Create 2021-06-30-join-demodam-hackathon.md

### PR DESCRIPTION
**Pull request by AlbaRoza**
New blogpost!

-----
[View rendered _posts/2021-06-30-join-demodam-hackathon.md](https://github.com/publiccodenet/blog/blob/Demodam/_posts/2021-06-30-join-demodam-hackathon.md)

### CODE REVIEW

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Join the Demodam hackathon
+
+Demodam is a new concept that brings together a community of developers, companies, municipalities and users. The idea of Demodam, a demo of a municipal website, enables reusable open source products and services available for municipalities in a single market place.
+
+## The hackathon
+
+During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here are the details you need to know.
+
+In [this collaborative document](https://docs.google.com/presentation/d/1ueLpnCIA06f05uuSTop8DU9olLb0M34iR_HrjsebRfI/edit#slide=id.p) you can find all the information about it. As in other [Common Ground](https://commonground.nl/cms/view/77953921-4161-462c-af24-6ac3cd7d61a7/common-ground-wat-hoe-en-waarom) hackathons, you just have to go to slide 17 and add your team name.
```
Comment (Ainali) [inline _posts/2021-06-30-join-demodam-hackathon.md:19]: This sounds strange (and perhaps it is!). Do you always have to go to slide 17 in Common Ground hackathons?

Comment (AlbaRoza) [reply _posts/2021-06-30-join-demodam-hackathon.md:19]: Not _that_ specific slide but it's the thing they've done as far as I know

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Join the Demodam hackathon"
+date: 2021-06-30
+author: Alba Roza
+type: blogpost
+excerpt: Information on how to join the following Demodam hackathon
+categories:
+  - Demodam
```
Comment (Ainali) [inline _posts/2021-06-30-join-demodam-hackathon.md:8]: Do we want a specific category for Demodam? I don't think we have one for any other codebase yet. I think we mostly use "news" instead.

Comment (AlbaRoza) [reply _posts/2021-06-30-join-demodam-hackathon.md]: Changed!

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Join the Demodam hackathon"
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:2]:
```suggestion
title: "Join the first Demodam hackathon, 5-9 July 2021"
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+date: 2021-06-30
+author: Alba Roza
+type: blogpost
+excerpt: Information on how to join the following Demodam hackathon
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:6]:
```suggestion
excerpt: Information on how to join the upcoming Demodam hackathon
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+categories:
+  - News
+---
+
+# Join the Demodam hackathon
+
+Demodam is a new concept that brings together a community of developers, companies, municipalities and users. The idea of Demodam, a demo of a municipal website, enables reusable open source products and services available for municipalities in a single market place.
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:13]: Does the idea of Demodam enable anything? Or Demodam itself?

```suggestion
Demodam is a new concept that brings together a community of developers, companies, municipalities and users. Demodam, a demo of a municipal website, showcases reusable open source products and services available for municipalities in a single market place.
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## The hackathon
+
+During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here are the details you need to know.
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:17]:
```suggestion
During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here's what you need to know.
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+In [this collaborative document](https://docs.google.com/presentation/d/1ueLpnCIA06f05uuSTop8DU9olLb0M34iR_HrjsebRfI/edit#slide=id.p) you can find all the information about it. As in other [Common Ground](https://commonground.nl/cms/view/77953921-4161-462c-af24-6ac3cd7d61a7/common-ground-wat-hoe-en-waarom) hackathons, you just have to go to slide 17 and add your team name.
+
+Once you select your favorite time slot for your pitch on the last day you'll just have to duplicate slide number 27 and edit it with your own characteristics. Some other competing teams have already added their proposals to the slide deck, we are waiting for yours!
+
+## Any doubts or comments?
+
+The website of Demodam is currently under development, so the best way of reaching out to the community is either:
+
+- Joining their #demodam [Slack channel](https://samenorganiseren.slack.com/join/shared_invite/zt-dex1d7sk-wy11sKYWCF0qQYjJHSMW5Q#/shared-invite/email)
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:27]:
```suggestion
- Joining the [#demodam Slack channel](https://samenorganiseren.slack.com/join/shared_invite/zt-dex1d7sk-wy11sKYWCF0qQYjJHSMW5Q#/shared-invite/email)
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+- Their [repository](https://github.com/demodam/demodam.org)
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:28]:
```suggestion
- [Checking out the repository](https://github.com/demodam/demodam.org)
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+- Reaching out to them [on Twitter](https://twitter.com/Demodam_cg)
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:29]:
```suggestion
- Reaching out [on Twitter](https://twitter.com/Demodam_cg)
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+- Following their updates [on LinkedIn](https://www.linkedin.com/company/demodam)
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:30]:
```suggestion
- Following their [LinkedIn updates](https://www.linkedin.com/company/demodam)
```

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here are the details you need to know.
+
+In [this collaborative document](https://docs.google.com/presentation/d/1ueLpnCIA06f05uuSTop8DU9olLb0M34iR_HrjsebRfI/edit#slide=id.p) you can find all the information about it. As in other [Common Ground](https://commonground.nl/cms/view/77953921-4161-462c-af24-6ac3cd7d61a7/common-ground-wat-hoe-en-waarom) hackathons, you just have to go to slide 17 and add your team name.
+
+Once you select your favorite time slot for your pitch on the last day you'll just have to duplicate slide number 27 and edit it with your own characteristics. Some other competing teams have already added their proposals to the slide deck, we are waiting for yours!
+
+## Any doubts or comments?
+
+The website of Demodam is currently under development, so the best way of reaching out to the community is either:
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:25]:
```suggestion
The Demodam website is currently under development, so meet the community by:
```

File: _posts/2021-06-30-join-demodam-hackathon.md (hunk 1 above)
Comment (ElenaFdR) [reply _posts/2021-06-30-join-demodam-hackathon.md:19]: Suggest removing the Common Ground link here - it muddies which link you need to follow to sign up. You could instead introduce CG support in the intro, including the link, if you think it's important to mention that context.

File: _posts/2021-06-30-join-demodam-hackathon.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+# Join the Demodam hackathon
+
+Demodam is a new concept that brings together a community of developers, companies, municipalities and users. The idea of Demodam, a demo of a municipal website, enables reusable open source products and services available for municipalities in a single market place.
+
+## The hackathon
+
+During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here are the details you need to know.
+
+In [this collaborative document](https://docs.google.com/presentation/d/1ueLpnCIA06f05uuSTop8DU9olLb0M34iR_HrjsebRfI/edit#slide=id.p) you can find all the information about it. As in other [Common Ground](https://commonground.nl/cms/view/77953921-4161-462c-af24-6ac3cd7d61a7/common-ground-wat-hoe-en-waarom) hackathons, you just have to go to slide 17 and add your team name.
```
Comment (ElenaFdR) [inline _posts/2021-06-30-join-demodam-hackathon.md:19]:
```suggestion
In [this collaborative document](https://docs.google.com/presentation/d/1ueLpnCIA06f05uuSTop8DU9olLb0M34iR_HrjsebRfI/edit#slide=id.p) you can find all the information about it - including goals, scope and the schedule. As in other [Common Ground](https://commonground.nl/cms/view/77953921-4161-462c-af24-6ac3cd7d61a7/common-ground-wat-hoe-en-waarom) hackathons, you just have to go to slide 17 and add your team name.
```

**Review by Ainali — DISMISSED** [review]
looks good to go!

File: _posts/2021-06-30-join-demodam-hackathon.md (hunk 6 above)
Comment (ElenaFdR) [reply _posts/2021-06-30-join-demodam-hackathon.md:17]: Suggest replacing "would you like to contribute" with a more concrete invitation - have you previously worked on a Common Ground component? Are you a dev (or whoever you're hoping to attract?) We'd love to have you!

**Review by ElenaFdR — COMMENTED** [review]
Some stylistic, some rhetorical suggestions.

**Review by Ainali — APPROVED** [review]
even better now!

**Pull request merged**